llvm/runtimes/CMakeLists.txt

```diff
@@ -270,13 +271,15 @@ function(runtime_default_target)
                                       -DLLVM_BUILD_TOOLS=${LLVM_BUILD_TOOLS}
                                       -DCMAKE_C_COMPILER_WORKS=ON
                                       -DCMAKE_CXX_COMPILER_WORKS=ON
+                                      -DCMAKE_Fortran_COMPILER_WORKS=ON
                                       -DCMAKE_ASM_COMPILER_WORKS=ON
                                       ${COMMON_CMAKE_ARGS}
                                       ${RUNTIMES_CMAKE_ARGS}
                                       ${ARG_CMAKE_ARGS}
                            PASSTHROUGH_PREFIXES LLVM_ENABLE_RUNTIMES
                                                 LLVM_USE_LINKER
-                                                CUDA # For runtimes that may look for the CUDA SDK (libc, offload)
+                                                CUDA # For runtimes that may look for the CUDA SDK (libc, offload, flang-rt)
+                                                FLANG_RUNTIME # Shared between Flang and Flang-RT
```

Comment:
We intentionally restrict the global prefixes, I think we check for the projects and then add the prefix instead of just doing it for everyone.

Reply:
This would be unrelated to this PR. `CUDA` is already passed unconditionally and has no effect if no runtime is using them. `FLANG_RUNTIME` is only used by `FLANG_RUNTIME_F128_MATH_LIB` that must be equal for Flang and Flang-RT. In the mid-term, we should strictly avoid any need to synchronize runtime and compiler configuration which are very problematic for cross-compilation and shared libraries. Once that is done, `FLANG_RUNTIME` can go away.